result/lldpneighbors.yaml

```diff
@@ -0,0 +1,155 @@
+components:
+  schemas:
+    LldpNeighbors.State.Request:
+      description: >-
+        The request to retrieve LLDP neighbor information for a given interface.
+      type: object
+      properties:
+        lldp_interface_names:
+          description: >-
+            The names of LLDP interfaces for which neighbor information will be
+            retrieved. If no names are specified then the results will
+            contain neighbor information for all configured LLDP interfaces.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.Lldp/properties/name"
+        neighbor_id_filters:
+          description: >-
+            Specify the neighbors for which information will be retuened. If the list is 
+            empty or missing then information for all neighbors will be returned.
+          type: array
+          items:
+            type: string
+    
+    LldpNeighbors.State:
+      description: >-
+        LLDP neighbor information.
+      type: object
+      properties:
+        lldp_name:
+          description: >-
+            The name of a LLDP interface.
+          type: string
+        system_name:
+          description: >-
+            The system name field shall contain an alpha-numeric string that 
+            indicates the system's administratively assigned name. The system name 
+            should be the system's fully qualified domain name. If implementations 
+            support IETF RFC 3418, the sysName object should be used for this field.
+          type: string
+        system_description:
+          description: >-
+            The system description field shall contain an alpha-numeric string that 
+            is the textual description of the network entity. The system description 
+            should include the full name and version identification of the system's 
+            hardware type, software operating system, and networking software. If 
+            implementations support IETF RFC 3418, the sysDescr object should be used 
+            for this field.
+          type: string
+        chassis_id:
+          description: >-
+            The Chassis ID is a mandatory TLV which identifies the chassis component of 
+            the endpoint identifier associated with the transmitting LLDP agent.
+          type: string
+        chassis_id_type:
+          description: >-
+            This field identifies the format and source of the chassis identifier string. 
+            It is an enumerator defined by the LldpChassisIdSubtype object from IEEE 802.1AB 
+            MIB.
+          type: string
+          enum: [port_component, network_address,chassis_component,mac_address,interface_name,local,interface_alias]
```

Comment:
Why are all the enums specified on one line instead of the usual convention that we have?

Reply:
in previously defined learned info all enums are like this,changing to 
enum:
         - def1
         - def2
is not working as port_component is present in 2 places and it doesn't compile,have to check why it is happening